plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -250,6 +251,29 @@ public void setColumnComment(SchemaTableName schemaTableName, String columnName,
         tableCache.invalidate(schemaTableName);
     }
 
+    public void renameTable(SchemaTableName oldName, SchemaTableName newName)
+    {
+        String oldSchemaName = toRemoteSchemaName(oldName.getSchemaName());
```

Comment:
I noticed we do not have tests for mongo utilizing: `BaseCaseInsensitiveMappingTest`, is there something preventing the mongo connector from utilizing these tests?

This would be helpful to verify the correctness of renames with case sensitive schemas/tables.

Reply:
We could utilize the test. The class doesn't contain test for `ALTER TABLE RENAME TO` though.